doc/modules/svm.rst

```diff
@@ -392,6 +392,11 @@ Tips on Practical Use
   * **Setting C**: ``C`` is ``1`` by default and it's a reasonable default
     choice.  If you have a lot of noisy observations you should decrease it.
     It corresponds to regularize more the estimation.
+    
+    :class:`LinearSVC` and :class`LinearSVR` are less sensitive to ``C`` when
+    ``C`` becomes large, and performance stays the same after a certain 
```

Comment:
But performance do you mean computational efficiency, or something about model accuracy?

Reply:
I meant model accuracy. The original paper mentions *"Once C is larger than certain value,
the obtained models have similar performances."* It can be contextually interpreted as that once C reaches a large enough value, increasing its value will not result in a more optimal model (i.e. better results).

Reply:
how about "results" or "predictions"?

Reply:
Yes that sounds better! E.g.

> `LinearSVC` and `LinearSVR` are less sensitive to ``C`` when it becomes large, and *prediction results stop improving* after a certain ...

Reply:
Please update the pr accordingly